values.yaml

```diff
@@ -250,25 +267,35 @@ jupyterhub:
   # ---- auth ----
   hub:
     config:
-      JupyterHub:
-        authenticator_class: ldapauthenticator.LDAPAuthenticator
-      LDAPAuthenticator:
-        bind_dn_template:
-          - 'cn={username},dc=ldap,dc=cetic,dc=be'
-          - 'cn={username},cn=admin,dc=ldap,dc=cetic,dc=be'
-        escape_userdn: false
-        lookup_dn: false
-        server_address: fadi-openldap
-        user_attribute: cn
-        use_ssl: false
+    # ---- Uncomment this blok to enable LDAP authentication ----
+      # JupyterHub:
+      #   authenticator_class: ldapauthenticator.LDAPAuthenticator
+      # LDAPAuthenticator:
+      #   bind_dn_template:
+      #     - 'cn={username},dc=ldap,dc=cetic,dc=be'
+      #     - 'cn={username},cn=admin,dc=ldap,dc=cetic,dc=be'
+      #   escape_userdn: false
+      #   lookup_dn: false
+      #   server_address: fadi-openldap
+      #   user_attribute: cn
+      #   use_ssl: false
+    # ---- Uncomment this blok to enable AzureAD authentication ----
+      # AzureAdOAuthenticator:
+      #   client_id: <your_application_id>
+      #   client_secret: <your_client_secret>
+      #   oauth_callback_url: https://<your_jupyterhub_domain_name>/hub/oauth_callback
+      #   tenant_id: <your_tenant_id>
+      # JupyterHub:
+      #   authenticator_class: azuread
   # ---- auth ----
   prePuller:
     hook:
       enabled: false
   # ---- ingress ----
   traefikIngress:
     enabled: true
-    host: hub.test.local
+    tls: false
+    host: <your_jupyterhub_domain_name>
```

Comment:
I would suggest to define all hosts (hub, grafana, adminer, ...) as this:

```
    host: jupyterhub.example.cetic.be  # change to your jupyterhub domainname
```

That way people can follow the install and userguide without having to change the values.yaml and redeploying.